src/client/components/VesselDetails.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from "react";
+import CardTeaser from "./CardTeaser";
+import { CardGroup, CardGroupHeader } from "./CardGroup";
+import VesselTeaser from "./VesselTeaser";
+
+function VesselDetailsComponent({
+  vesselName,
+  departureFrom,
+  arrivesAt,
+  etd,
+  eta,
+  date,
+  lat,
+  lon,
```

Comment:
Please rename to `longitude` and `latitude` for constistency :)

Reply:
And remember to replace throughout the PR.

Reply:
renamed to longitude and latitude